app/views/shared/processed_and_deleted/_show.html.slim

```diff
@@ -7,7 +7,7 @@ header
 =build_section_with_defaults 'Personal details', @applicant
 =build_section_with_defaults 'Application details', @details
 =build_section 'Benefits', @application_view, %w[benefits]
-=build_section 'Income', @application_view, %w[number_of_children total_monthly_income]
+=build_section 'Income', @application_view, %w[number_of_children total_monthly_income total_monthly_income_from_evidence]
```

Comment:
I thought we all hated %w's ?

Reply:
@garytaylor I guess this was not refactored. I can create a ticket for refactoring these.